BUILD.bazel

```diff
@@ -54,29 +54,29 @@ nogo(
     # that are always correct).
     # You can see the what `go vet` does by running `go doc cmd/vet`.
     deps = [
-        "//vendor/github.com/gordonklaus/ineffassign/pkg/ineffassign:go_tool_library",
-        "@org_golang_x_tools//go/analysis/passes/asmdecl:go_tool_library",
-        "@org_golang_x_tools//go/analysis/passes/assign:go_tool_library",
-        "@org_golang_x_tools//go/analysis/passes/atomic:go_tool_library",
-        "@org_golang_x_tools//go/analysis/passes/bools:go_tool_library",
-        "@org_golang_x_tools//go/analysis/passes/buildtag:go_tool_library",
+        #"//vendor/github.com/gordonklaus/ineffassign/pkg/ineffassign:go_default_library",
```

Comment:
`ineffassign` isa custom validator which we added. It still uses `go_tool_library` like the other nogo validators did. You can change it rules in `hack/bazel-generate.sh` to also have a `go_default_library` target now and that it uses `go_library` instead of `go_tool_library`.